packages/google_maps_flutter/google_maps_flutter_web/example/web/index.html

```diff
@@ -6,7 +6,7 @@
     <head>
         <title>Browser Tests</title>
         <!-- This API key comes from: go/flutter-maps-web-tests-api-key (GCP project: flutter-infra) -->
-        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAa9cRBkhuxGq3Xw3HPz8SPwaVOhRmm7kk&libraries=geometry"></script>
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAa9cRBkhuxGq3Xw3HPz8SPwaVOhRmm7kk&libraries=geometry,visualization"></script>
```

Comment:
IIRC `geometry` is only required for the `projection_test.dart`. I think you should be able to remove `visualization` from here?

Reply:
Did you mean to say I can remove `geometry`?

Reply:
There is code in the integration test that uses heatmaps so I'm not sure if either of them can be removed

Reply:
`geometry` must stay, because we actually use it to compute boundaries within a really-rendered map (see `projection_test.dart`). Not sure about `visualization`, but this is a good example of using 2 libraries, so let's leave this in!